app/src/main/java/org/eyeseetea/malariacare/data/database/model/Survey.java

```diff
@@ -989,6 +1009,85 @@ public SurveyAnsweredRatio reloadSurveyAnsweredRatio() {
         return surveyAnsweredRatio;
     }
 
+    private SurveyAnsweredRatio reloadStockSurveyAnsweredRatio() {
```

Comment:
@manuelplazaspalacio DBFlow survey model is a class with a lot of responsibilities. All the new code that we do is better to separate it and little by little leave here only the columns of the table. I think it would be possible to add the logic of calculating the ratio of questions answered in domain and not in the data layer. So the new code is easier to split into methods to make it cleaner.